pkgs/by-name/pa/paper-plane/rlottie.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, meson
+, ninja
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rlottie-paperplane";
+  version = "unstable-2022-09-14";
```

```suggestion
  version = "0-unstable-2022-09-14";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming